bindings/node.js/kzg.cxx

```diff
@@ -173,31 +224,45 @@ Napi::Value ComputeKzgProof(const Napi::CallbackInfo& info) {
   KZGProof proof;
   C_KZG_RET ret = compute_kzg_proof(
     &proof,
-    (Blob *)blob,
-    (Bytes32 *)z_bytes,
+    blob,
+    z_bytes,
     kzg_settings
   );
 
   if (ret != C_KZG_OK) {
      Napi::Error::New(env, "Failed to compute proof")
       .ThrowAsJavaScriptException();
-    return env.Undefined();
+    return env.Null();
   };
 
-  return napi_typed_array_from_bytes((uint8_t *)(&proof), BYTES_PER_PROOF, env);
+  return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<uint8_t *>(&proof), BYTES_PER_PROOF);
 }
 
-// computeBlobKzgProof: (blob: Blob, setupHandle: SetupHandle) => KZGProof;
-Napi::Value ComputeBlobKzgProof(const Napi::CallbackInfo& info) {
-  auto env = info.Env();
 
+/**
+ * Given a blob, return the KZG proof that is used to verify it against the
+ * commitment.
+ * 
+ * @param {Blob} blob - The blob (polynomial) to generate a proof for
+ * 
+ * @returns {KZGProof} - The resulting proof
```

Comment:
Nit, noticed mixed usage between `returns` and `return` docs. I think we should stick to one style. I prefer `returns` but not a strong opinion, would be fine with either.

Reply:
I found `return` in the `c_kzg_4844.c` and matched that